frontend/src/MovieGallery/model/PopularMovie.ts

```diff
@@ -0,0 +1,15 @@
+export type PopularMovie = {
```

Comment:
Do you actually need the same data structure 3 times? 🤔

Reply:
its not the same ;-)